partcad/src/partcad/sketch.py

```
@@ -11,18 +11,20 @@
 
 from .shape_ai import ShapeWithAi
 from . import sync_threads as pc_thread
+from .user_config import UserConfig
 
 
 class Sketch(ShapeWithAi):
     path: typing.Optional[str] = None
 
-    def __init__(self, project_name: str, config: dict = {}) -> None:
-        super().__init__(project_name, config)
+    def __init__(self, project_name: str, config: dict = {}, user_config: UserConfig = None) -> None:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)